docs/developer_guide/getting_started.md

```diff
@@ -1,67 +1,46 @@
 
 ## Create a new Node.js project
 
 1. Create and enter a new directory to hold the project:
+  ### Linux/Mac
+  1. `mkdir ~/offline_wallet_generator`
+  2. `cd ~/offline_wallet_generator`
+
   ### Windows
   1. `mkdir C:\offline_wallet_generator`
   2. `cd C:\offline_wallet_generator`
 
-  ### Linux
-  1. `mkdir ~/offline_wallet_generator`
-  2. `cd ~/offline_wallet_generator`
-  
 2. Create the new project:
   `npm init`
-  npm will ask you to enter information into a number of fields. Press enter at each prompt to use the default (or empty) value.
+  The npm setup utility will ask you for some information. Press enter at each prompt to use the default value.
 
 3. Add the monero-javascript library to the offline_wallet_generator package:
   `npm install monero-javascript`
 
-# Write a monero-javascript program
-## Creating an offline wallet generator
+## Write a monero-javascript program
+### Creating an offline wallet generator
 
-An offline wallet generator creates and displays a new, random wallet address along with that address's associated view key, spend key, and mnemonic seed phrase. Offline wallet generators only need to generate and display these wallet attributes; they do not need to communicate with a Monero network, transfer XMR or track a wallet's balance or outputs.
+An offline wallet generator creates and displays a new, random wallet address along with that address's associated view key, spend key, and mnemonic seed phrase. An offline wallet generator only needs to generate and display these wallet attributes. It does not need to communicate with a Monero network, transfer XMR or track a wallet's balance or outputs.
 
-Monero-javascript's keys-only wallet more than meets these requirements and is the ideal basis for a monero-javascript-based offline wallet generator. They keys-only wallet is a minimal Wasm wallet implementation that can not initiate transfers, report its balance, or communicate with a Monero network. The trade off for these limitations is a small file size - just under 1/5 that of a standard Wasm wallet. These characteristics make it the ideal basis for an offline wallet.
+Monero-javascript's keys-only wallet more than meets these requirements and is the ideal basis for a monero-javascript-based offline wallet generator. The monero-javascript wallet provides a minimal Wasm wallet implementation, called a keys-only wllet, that tracks only the wallet's _permanent_ attributes. It can not initiate transfers, report its balance, or communicate with a Monero network. The trade off for these limitations is a small file size - just under 1/5 that of a standard Wasm wallet. The smaller memory footprint allows programs to load more quickly an run more efficiently. Therefore, you should use keys-only wallets whenever your program does not require a full Wasm or RPC wallet.
```

Comment:
`The trade off for these limitations is a small file size`
Small file size must be a benefit, not a trade off.